content/teknologi/altinnstudio/changelog/app-nuget/v4/whats-new.md

```diff
@@ -5,10 +5,34 @@ description: Overview of changes introduced in version 4
 toc: true
 ---
 
+### 4.7.0 (2021-06-08)
+
+Altinn Apps now authorize access for statless apps.
+
+Altinn Apps have now two new Application Events where application developers can add data processing logic. Calulation, population +++
```

Comment:
Instead of +++ we could write "and more", or something similar.
```suggestion
Altinn Apps now have two new application events where application developers can add data processing logic. calulation, population, and more.
```